src/components/projectSelect/ProjectOptionGroup.tsx

```diff
@@ -37,9 +39,19 @@ export const ProjectOptionGroup: React.FC<OptionGroupProps> = ({
   name,
   projects,
   onClick,
+  repoIdentifier,
+  isProjectSettingsPage = false,
 }) => (
   <OptionGroupContainer>
-    <Overline>{name}</Overline>
+    {/* if it's the project settings page and it's not the "" group, make the header clickable */}
+    {repoIdentifier === "" || !isProjectSettingsPage ? (
+      <Overline>{name} </Overline>
+    ) : (
+      <OverlineHover>
+        <Overline onClick={() => onClick(repoIdentifier)}>{name} </Overline>
```

Comment:
Can you add a `role="button"` To help the browser know this is a clickable button. https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role

Reply:
Usually the linter would warn you about this but since its a styled element it can't really tell its a div.

Reply:
done